app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    separate = command.split()
+    old_file = separate[1]
+    new_file = separate[2]
+    if old_file == new_file:
```

Comment:
Make sure you check that the cp input command is correct and that the command consists of three parts